### PR TITLE
Allow any XColor values as --image-bg argument

### PR DESCRIPTION
**Pull request by ulteq**
I tried my best to retain the old default behavior.

Loosely based on: https://github.com/derf/feh/pull/126

Closes: https://github.com/derf/feh/issues/125
Closes: https://github.com/derf/feh/pull/126

### CODE REVIEW

File: src/wallpaper.c [hunk 1]
```diff
@@ -305,15 +305,19 @@ void feh_wm_set_bg(char *fil, Imlib_Image im, int centered, int scaled,
 
 		D(("Falling back to XSetRootWindowPixmap\n"));
 
+		XColor color;
+		Colormap cmap = DefaultColormap(disp, DefaultScreen(disp));
+		if (opt.image_bg)
```
Comment (Ferada) [inline src/wallpaper.c]: Could `image_bg` default to `"black"` actually to remove this branch?

File: src/help.raw [hunk 2]
```diff
@@ -84,7 +84,7 @@ OPTIONS
                            can be used multiple times to add multiple paths.
  -M, --menu-font FONT      Use FONT for the font in menus.
  -B, --image-bg STYLE      Set background for transparent images and the like.
-                           Accepted values: white, black, default
+                           Accepted values: default, checks, or a XColor (eg. #428bdd)
```
Comment (Ferada) [inline src/help.raw]: Old code accepted `"checks"`, `"white"` and `"black"` (contrary to the manual), the new one actually does accept `"default"` - if it wasn't before, maybe leave it out here and default to `"black"` as mentioned below?

Comment (ulteq) [reply src/help.raw]: The old code worked like that:

| | "white"  | "black" | "checks" | [anything else] |
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Windowed  | White  | Black | Checks | Checks |
| Fullscreen  | White  | Black | Checks | Black |

The new code works the same, but [anything else] is now either no `--image-bg` at all, or `--image-bg default`. Any other string is converted to a color, which is then used for both windowed and fullscreen mode.

File: src/wallpaper.c (hunk 1 above)
Comment (ulteq) [reply src/wallpaper.c]: We can do it like that, if we drop the old 'default' behavior:
> The default for windowed mode is checks, while fullscreen defaults to black.

File: src/help.raw (hunk 2 above)
Comment (Ferada) [reply src/help.raw]: Okay, then never mind my comments, looking great.

**Pull request merged**